campaign-launcher/interfaceV2/src/utils/TokenManager.ts

```diff
@@ -0,0 +1,65 @@
+import { decodeJwt } from '.';
+import { ACCESS_TOKEN_KEY, REFRESH_TOKEN_KEY } from '../constants';
```

Comment:
Maybe defined these constants in this file? I don't think they might/should be used anywhere else